uiServices/src/homepage.go

```diff
@@ -37,7 +37,7 @@ func GetIndexPage() *chi.Mux {
 	router.Get("/", func(w http.ResponseWriter, r *http.Request) {
 
 		goPath := os.Getenv("GOPATH")
-		absPath, err := filepath.Abs(goPath + "/src/github.com/xtracdev/automated-perf-test/ui/index.html")
+		absPath, err := filepath.Abs(goPath + "/src/github.com/xtracdev/automated-perf-test/ui-src/src/index.html")
```

Comment:
Is there a reason why you changed to point to the precompiled index.html

Reply:
Sorry, Didn't know there was 2 index.html pages. Knew there was one at "ui-src/src/index.html". Thought it was at error pointing at wrong location "/ui/index.html".  I'll revert it now!